webknossos/tests/dataset/test_add_layer_from_images.py

```diff
@@ -461,34 +499,49 @@ def test_test_images(
         assert l_normal.num_channels == num_channels
         assert l_normal.bounding_box.size.to_tuple() == size
         if l_bio is not None:
-            assert np.array_equal(
+            np.testing.assert_array_equal(
                 l_bio.get_finest_mag().read(), l_normal.get_finest_mag().read()
             )
     return ds
 
 
+@pytest.mark.parametrize(
+    "url, filename, kwargs, dtype, num_channels, size", TEST_IMAGES_ARGS
+)
+def test_test_images(
+    tmp_path: Path,
+    url: Union[str, List[str]],
+    filename: Union[str, List[str]],
+    kwargs: Dict,
+    dtype: str,
+    num_channels: int,
+    size: Tuple[int, int, int],
+) -> None:
+    _test_test_images(tmp_path, url, filename, kwargs, dtype, num_channels, size)
```

Comment:
What was the purpose of adding this new method and just giving it all arguments that are already present in this test method?

Reply:
pytest complained about test functions having a non-None return value.

Reply:
Alternatively, we can remove the functionality to call these modules as CLI tools.